components/recurring-contributions/RecurringContributionsPopUp.js

```diff
@@ -82,7 +82,7 @@ const cancelRecurringContributionMutation = gql`
   }
 `;
 
-const RecurringContributionsPopUp = ({ contribution, status, onCloseEdit, account }) => {
+const RecurringContributionsPopUp = ({ contribution, status, onCloseEdit, account, LoggedInUser }) => {
```

Comment:
:ice_cream: (no requested changes) Since it's a functional component, it would be better to rely on `useUser` rather than `withUser`

Reply:
Thanks for pointing that out. Maybe we need to refactor this in all places later, cause it seems that we do this many different places. 🤔